# Coinfection by different Dengue clades in Southern Brazil
Supplementary material and data used in the analysis of the manuscript entitlet "Simultaneous circulation and coinfection of Dengue virus serotypes and clades in Southern Brazil".

See our preprint at https://www.biorxiv.org/content/10.1101/2024.02.24.581892v1.article-info

DATA AND MATERIALS AVAILABILITY
- Datasets from figures 1, figure 2A&B and supplementary figure 1A&B;
- Metrics from intrahost variation analysis and genome assembly from coinfection cases;
- Supplementary material;
- Raw reads and fastas MajV/MinV from coinfection cases.

Supplementary Figure 1: A. Maximum likelihood tree for complete dataset DENV-1. The six clades identified in our study are highlighted, as shown in the legend below the tree; B. DENV-2 maximum likelihood tree for the complete dataset. The two clades identified in our study are highlighted, as shown in the legend below the tree.

Supplementary Figure 2: Plot with the frequencies of the two well-supported alleles for the MajV and MinV genomes of coinfection cases: Denv1_1985_2022, Denv1_2010_2022, Denv1_7052_2022 and Denv1_8625_2022. On the X axis we have the position in the genome and on the Y axis, we have the frequency in percentage over the depth of reads in a given allele. The part of the bar in blue represents the frequency of the MajV allele and the part of the bar in red represents the frequency of the MinV allele in percentage. The nucleotide that represents this frequency in each position is positioned in the center of the bar.

Supplementary Figure 3. Plot with iSNV values for complete genomes of all samples, highlighting the four outlier samples with a notable gap between the values.

Supplementary Table 1. iSNV values for complete genomes of all samples. Contain in the first column all 627 DENV1 samples followed by their iSNV values and all 40 DENV2 samples followed by their iSNV values.  Available at: Supplementary/Denv_iSNV_allsamples.xlsx.
